components/brave_wallet_ui/components/extension/shared-panel-styles.ts

```diff
@@ -95,3 +97,44 @@ export const TabRow = styled.div`
   width: 255px;
   margin-bottom: 10px;
 `
+
+export const StatusBubble = styled.div<Partial<StyleProps>>`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 10px;
+  height: 10px;
+  border-radius: 100%;
+  opacity: ${(p) => p.status === 3 || p.status === 1 || p.status === 0 ? 0.4 : 1};
+  background-color: ${(p) => p.status === 4 || p.status === 1
+    ? '#2AC194'
+    : p.status === 2 || p.status === 5 ? '#EE6374'
+      : p.status === 0 ? p.theme.color.interactive08 : p.theme.color.warningIcon
```

Comment:
Two remarks:
1. Can we use proper types instead of comparing integers like `p.status === 2`?
2. Can `#EE6374` be imported from the theme? I guess not, but probably a good idea to create an issue for including it in `brave-ui`?

Reply:
Updated!
Also updated the color theme to what is in Figma!